src/queries/placeholderText.ts

```diff
@@ -0,0 +1,105 @@
+import type { ReactTestInstance } from 'react-test-renderer';
```

Comment:
I like the new simpler naming!

we should also rename the files in the __test__ folders, wdyt ?

Reply:
maybe even make a queries subfolder in the test folder

Reply:
totally! if we do that, shouldn't we also do it for the helpers? (to stay cohesive?)

Reply:
moved queries tests to `src/queries/__tests__` directory.